src/main/webapp/app/css/calendar.css

```diff
@@ -0,0 +1,158 @@
+body {
+    background: #e0e0e0
+}
+
+#cal {
+    -moz-box-shadow: 0 3px 3px rgba(0, 0, 0, .25);
+    -webkit-box-shadow: 0 3px 3px rgba(0, 0, 0, .25);
+    margin: 50px auto;
+    font: 13px/1.5 helvetica neue, Helvatica, Arial, san-serif;
+    display: table;
+    position: absolute
+}
+
+#cal .header {
+    cursor: default;
+    background: #cd310d;
+    background: -moz-linear-gradient(top, #b32b0c, #cd310d);
+    background: -webkit-gradient(linear, left top, left bottom, from(#b32b0c), to(#cd310d));
+    height: 34px;
+    position: relative;
+    color: #fff;
+    -webkit-border-top-left-radius: 5px;
+    -webkit-border-top-right-radius: 5px;
+    -moz-border-radius-topleft: 5px;
+    -moz-border-radius-topright: 5px;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+    font-weight: 700;
+    text-shadow: 0 -1px 0 #87260c;
+    text-transform: uppercase
+}
+
+#cal .header span {
+    display: inline-block;
+    line-height: 34px
+}
+
+#cal .header .hook {
+    width: 9px;
+    height: 28px;
+    position: absolute;
+    bottom: 60%;
+    border-radius: 10px;
+    -moz-border-radius: 10px;
+    -webkit-border-radius: 10px
+}
+
+.right.hook {
+    right: 15%
+}
+
+.left.hook {
+    left: 15%
+}
+
+#cal .header .button {
+    width: 24px;
+    text-align: center;
+    position: absolute
+}
+
+#cal .header .left.button {
+    left: 0;
+    -webkit-border-top-left-radius: 5px;
+    -moz-border-radius-topleft: 5px;
+    border-top-left-radius: 5px;
+    border-right: 1px solid #ae2a0c
+}
+
+#cal .header .right.button {
+    right: 0;
+    top: 0;
+    border-left: 1px solid #ae2a0c;
+    -webkit-border-top-right-radius: 5px;
+    -moz-border-radius-topright: 5px;
+    border-top-right-radius: 5px
+}
+
+#cal .header .button:hover {
+    background: -moz-linear-gradient(top, #d94215, #bb330f);
+    background: -webkit-gradient(linear, left top, left bottom, from(#d94215), to(#bb330f))
```

Comment:
Would be nice to have color names instead of codes. There are quite a lot of supported names - e.g. here:
https://www.rapidtables.com/web/css/css-color.html
https://www.w3schools.com/colors/colors_names.asp